toonz/sources/common/tsystem/tsystempd.cpp

```diff
@@ -385,6 +436,31 @@ TINT64 TSystem::getMemorySize(bool onlyPhisicalMemory) {
   free(sysInfo);
   return ret;
 
+#elif defined(FREEBSD)
+
+  TINT64 ret = 0;
+#ifdef __OpenBSD__
+  int mib[] = {CTL_VM, VM_UVMEXP};
+  struct uvmexp  uvmexp;
+#else
+  int mib[] = {CTL_VM, VM_TOTAL};
+  struct vmtotal vmtotal;
+#endif
+  size_t size;
```

Comment:
Push this variable definition to the top of the block, otherwise it's quite confusing.